tools/pipeline-generator/docs/design.md

```diff
@@ -0,0 +1,239 @@
+# **MONAI Bundle Integration for MONAI Deploy App SDK & Holoscan SDK**
+
+## **Objective**
+
+The goal of this project is to build a robust tool that enables a seamless path for developers to integrate AI models exported in the **MONAI Bundle format** into inference-based applications built with the **MONAI Deploy App SDK** and the **Holoscan SDK**.
+
+This tool will support:
+
+- Standard MONAI Bundles (.pt, .ts, .onnx)
+- MONAI Bundles exported in **Hugging Face-compatible format**
+
+By bridging the gap between model packaging and application deployment, this project aims to simplify clinical AI prototyping and deployment across NVIDIA’s edge AI platforms.
+
+## **Background**
+
+The **MONAI Bundle** is a standardized format designed to package deep learning models for medical imaging. It includes the model weights, metadata, transforms, and documentation needed to make the model self-contained and portable.
+
+The **Holoscan SDK** is NVIDIA’s real-time streaming application SDK for AI workloads in healthcare and edge devices. The **MONAI Deploy App SDK** is designed for building composable inference applications, often used in radiology and imaging pipelines.
```

Comment:
May want to explicitly state that MD App SDK is based on HSDK with added medical image domain specific operarator/functionalities.